wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -4,12 +4,11 @@
 {% block bodyclass %}page-explorer {% if ordering == 'ord' %}reordering{% endif %}{% endblock %}
 
 {% block content %}
-    <header class="w-header w-header--merged w-header--no-border">
-        <h1 class="visuallyhidden">Explorer</h1>
-
-        {% breadcrumbs parent_page 'wagtailadmin_explore' url_root_name='wagtailadmin_explore_root' %}
-    </header>
-
+    {% page_permissions parent_page as page_perms %}
```

Comment:
This is also done on line 16, `{% page_permissions parent_page as parent_page_perms %}`. Could we only do this once? I assume it’d result in better performance for us to only do one query for this.